src/HoC.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {ensure} from './util';
+import {update, unmount} from './Mixin';
+
+// Mixin for RethinkDB query subscription support in React components. You'll
+// generally want to use DefaultHoC or PropsHoC, which use BaseHoC to
+// create more usable versions.
+//
+// In your component, you should define an observe(props, state) function that
+// returns an object mapping query names to QueryRequests. See
+// QueryRequest.js for the API.
```

Comment:
I think this comment might be misleading, since the client shouldn't define `observe` in their component, but rather should pass in the `observe` function to the HoC.